tests/integration/crud/advanced/flow-index.yml

```diff
@@ -0,0 +1,20 @@
+!Flow
+with:
+  restful: $RESTFUL
```

Comment:
no need to parametrize restful imho, we are testing a feature that should not test restful capabilities

Reply:
thanks, I removed it